pkgs/development/python-modules/django-simple-captcha/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, buildPythonPackage, mock, 
+django, django-ranged-response, pillow }:
+
+buildPythonPackage rec {
+  pname = "django-simple-captcha";
+  version = "0.5.6";
+
+  src = fetchurl {
```

Comment:
Since the package is available on pypi, I think `fetchPypi` should be prefered.

Reply:
I tried PyPi, but there's no `.tar.gz` file available for `0.5.6` and I wasn't able to build from the `.whl` yet :(